samples/docker_aks/Makefile

```diff
@@ -34,7 +34,7 @@ $(PACKAGE): $(APPDIR)
 	./$(PACKAGE) red green blue
 
 run: $(APPDIR)
-	@docker run --rm --device=/dev/sgx:/dev/sgx --volume /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket $(DOCKER_IMG)
+	@docker run --rm --device=/dev/sgx/enclave:/dev/sgx/enclave --volume /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket $(DOCKER_IMG)
```

Comment:
Better to update to /dev/sgx_enclave now, and to include /dev/sgx_provision for provisioning enclaves.